pages/docs/tutorials/validate.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Validate AsyncAPI Files"
+description: In this tutorial we'll learn how to validate AsyncAPI files.
+weight: 120
+---
+
+# Introduction
+We'll discuss what it means to validate AsyncAPI files in this tutorial. AsyncAPI files can be validated in two different ways; by validating the AsyncAPI document and by validating messages or events in runtime. 
+
+This tutorial's main focus will be on how AsyncAPI document validation is done.
+
+# Background context
+
+AsyncAPI is an open-source specification standard that seeks to improve the current state of Event-Driven Architecture (EDA) by making it as easy to work with as REST APIs. That goes from documentation to code generation, from discovery to event management. 
+
+With AsyncAPI, you can create specifications that allows developers, architects, and product managers to define the interfaces of an event API the same way, [OpenAPI (formerly known as Swagger)](https://github.com/OAI/OpenAPI-Specification) does for REST APIs.
+
+
+# Installation Guide
+Before you proceed to the next stage, you'll need to download a few things:
+
+1. Install [Node.js](https://nodejs.org/en/download/) (v15 or newer).
+2. Install Git on your Operating System (OS):
+  - [Install Git on MacOs](https://git-scm.com/download/mac)
+  - [Install Git on Windows](https://git-scm.com/download/win)
+  - [Install Git on Linux](https://git-scm.com/download/linux)
+
+# Creating an AsyncAPI Document
```

Comment:
```suggestion
# Create an AsyncAPI Document
```

I think it should be `Create`, you basically tell the user, that they need to create asyncapi document